test/e2e/json-rpc/wallet_getCapabilities.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { strict as assert } from 'assert';
+import { withFixtures } from '../helpers';
+import FixtureBuilder from '../fixture-builder';
+import TestDapp from '../page-objects/pages/test-dapp';
+import { loginWithBalanceValidation } from '../page-objects/flows/login.flow';
+import { DEFAULT_FIXTURE_ACCOUNT } from '../constants';
+import { mockEip7702FeatureFlag } from '../tests/confirmations/helpers';
+
+describe('wallet_getCapabilities', function () {
+  it('should indicate auxiliaryFunds support for chains with bridge support', async function () {
```

Comment:
should we also have tests that test the account support dimension?

`should indicate auxiliaryFunds support for non hardware wallet accounts`? etc?